web/src/search/results/SearchResultsFilterBars.tsx

```diff
@@ -61,36 +85,40 @@ export const SearchResultsFilterBars: React.FunctionComponent<{
                 </div>
             </div>
         )}
-        {isSearchResults(results) && results.dynamicFilters.filter(filter => filter.kind === 'repo').length > 0 && (
-            <div className="search-results-filter-bars__row" data-testid="repo-filters-bar">
-                Repositories:
-                <div className="search-results-filter-bars__filters">
-                    {results.dynamicFilters
-                        .filter(filter => filter.kind === 'repo' && filter.value !== '')
-                        .map((filter, i) => (
+        {isSearchResults(results) &&
+            results.dynamicFilters
+                .filter(filter => filter.kind === 'repo')
+                .filter(filter => !isScopeSelected(navbarSearchQuery, filter.value)).length > 0 && (
+                <div className="search-results-filter-bars__row" data-testid="repo-filters-bar">
+                    Repositories:
+                    <div className="search-results-filter-bars__filters">
+                        {results.dynamicFilters
+                            .filter(filter => filter.kind === 'repo' && filter.value !== '')
+                            .filter(filter => !isScopeSelected(navbarSearchQuery, filter.value))
+                            .map((filter, i) => (
+                                <FilterChip
+                                    name={filter.label}
+                                    isSelected={isScopeSelected(navbarSearchQuery, filter.value)}
```

Comment:
```suggestion
                                    isSelected={false}
```

`results.dynamicFilters` are filtered with `filter => !isScopeSelected(navbarSearchQuery, filter.value)` above.